mkt/detail/tests/test_views.py

```diff
@@ -120,20 +114,20 @@ def test_app_unlisted_author(self, _mock):
     def test_app_private(self):
         self.app.update(status=mkt.STATUS_APPROVED)
         res = self.client.get(self.url)
-        eq_(res.status_code, 404)
+        eq_(res.status_code, 403)
```

Comment:
I'm not sure if I like that a `403` could give a hint that the manifest exists but the user doesn't have access to it, but really, this is the manifest URL with the UUID. We should be more concerned about this on the app detail API but we're also returning 403s there so this probably doesn't matter much. It is more correct.